INSTALL.md

```diff
@@ -27,6 +27,11 @@ DEPENDENCY
 Now uftrace can be built without any external libraries.  But in order to use
 more advanced features, it'd be better to install them like below.
 
+Firstly, please make sure `pkg-config` is installed in the system to properly
+detect the dependencies for uftrace. Otherwise, some packages may not be
```

Comment:
Sorry for pointing out once again but let me change two more.
1. "dependencies for uftrace" -> "dependencies of uftrace"
2. we prefer to have two white spaces between two sentences, so please add one more space before "Otherwise".

Reply:
@honggyukim I changed. Thanks for your guide.

Reply:
Looks good. Thanks for your work. I think @namhyung will merge this soon.